docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -16,6 +16,21 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-virtual-background      | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background)           |
 
 
+## 28-11-2024
+
+Released: `@100mslive/hms-video-store@0.12.25`, `@100mslive/react-sdk@0.10.25`, `@100mslive/hls-player@0.3.25`, `@100mslive/roomkit-react@0.3.25`, `@100mslive/hms-whiteboard@0.0.15`, `@100mslive/hms-virtual-background@1.13.25`
+
+### Breaking
+
+- Implemented internal method to check if recording should continue when no active elements are present
+
+### Fixed
+- Removed autohiding of Top and Bottom Navigation Bar controls for Mobile Browsers
+- Persisting selection of audio input/output & video input device across local peer's role change
+- Fixed issue where PIP Chat window's height was not being calculated correctly
```

Comment:
This is specific to Chrome 131. Also let's prefix prebuilt changes with roomkit prebuilt so people can differentiate between prebuilt fixed and SDK fixes

Reply:
Done